lib/at_cmd/at_cmd.c

```diff
@@ -145,6 +147,7 @@ static void socket_thread_fn(void *arg1, void *arg2, void *arg3)
 
 		bytes_read = recv(common_socket_fd, item->data,
 				  sizeof(item->data), 0);
+
```

Comment:
Empty line not needed.

Reply:
Because of the linebreak, it's easier to read with a empty line between.